restler/engine/core/request_utilities.py

```diff
@@ -50,56 +53,95 @@ def str_to_hex_def(val_str):
     """
     return hashlib.sha1(val_str.encode(UTF8)).hexdigest()
 
-def execute_token_refresh_cmd(cmd):
-    """ Forks a subprocess to execute @param cmd to refresh token.
-
-    @param cmd: The user-provided command to refresh the token.
-    @type  cmd: Str
-
-    @return: The result of the command
-    @rtype : Str
 
-    """
+def execute_token_refresh(token_dict):
```

Comment:
>execute_token_refresh [](http://example.com/codeflow?start=4&length=21)

Please add header comments to these functions

Reply:
adding these